boilerplate/app/components/Toggle/Switch.tsx

```diff
@@ -38,7 +38,16 @@ interface SwitchInputProps extends BaseToggleInputProps<SwitchToggleProps> {
  * @returns {JSX.Element} The rendered `Switch` component.
  */
 export function Switch(props: SwitchToggleProps) {
-  return <Toggle accessibilityRole="switch" {...props} ToggleInput={SwitchInput} />
+  const { accessibilityMode, ...rest } = props
+  return (
+    <Toggle
+      accessibilityRole="switch"
+      {...rest}
+      ToggleInput={(toggleProps) => (
+        <SwitchInput {...toggleProps} accessibilityMode={accessibilityMode} />
+      )}
+    />
```

Comment:
These inputs need to be in a callback to preserve the input identity so that animations work correctly, e.g., 

```

/**
 * @param {SwitchToggleProps} props - The props for the `Switch` component.
 * @returns {JSX.Element} The rendered `Switch` component.
 */
export function Switch(props: SwitchToggleProps) {
  const { accessibilityMode, ...rest } = props
  const toggleInput = useCallback(
    (toggleProps: SwitchInputProps) => (
      <SwitchInput {...toggleProps} accessibilityMode={accessibilityMode} />
    ),
    [accessibilityMode],
  )
  return <Toggle accessibilityRole="switch" {...rest} ToggleInput={toggleInput} />
}
```